python/pyspark/sql/functions.py

```diff
@@ -189,15 +210,15 @@ def _():
 }
 
 for _name, _doc in _functions.items():
-    globals()[_name] = since(1.3)(_create_function(_name, _doc))
+    globals()[_name] = since(1.3)(ignore_unicode_prefix(_create_function(_name, _doc)))
```

Comment:
Hm, this then applies `ignore_unicode_prefix` to all docs ... This looks only added for `_lit_doc`. Could we simply avoid the unicode example in the lit doc rather than applying this to all function docs?